beamer/agent/commands.py

```diff
@@ -0,0 +1,127 @@
+import signal
+import sys
+from importlib.metadata import version
+from pathlib import Path
+from typing import Optional
+
+import click
+import structlog
+
+import beamer.agent.config
+import beamer.agent.contracts
+import beamer.agent.util
+from beamer.agent.agent import Agent
+from beamer.agent.l1_resolution import get_relayer_executable
+
+log = structlog.get_logger(__name__)
+
+
+def _sigint_handler(agent: Agent) -> None:
+    signal.signal(signal.SIGINT, signal.SIG_IGN)
+    log.info("Received SIGINT, shutting down")
+    agent.stop()
+
+
+@click.command("agent")
```

Comment:
I would name this file just `command.py` instead of `commands.py` because there is only going to be one "beamer agent" command, the other things in here are options. Similarly for `monitoring`.

Reply:
It's one right now. But what if we add:
`beamer health-check agent-balance AGENT_ADDRESS`

tomorrow?